Sources/SnapshotTesting/Common/View.swift

```diff
@@ -941,6 +941,10 @@ private func add(traits: UITraitCollection, viewController: UIViewController, to
 
   return {
     rootViewController.beginAppearanceTransition(false, animated: false)
+    viewController.willMove(toParent: nil)
+    viewController.view.removeFromSuperview()
+    viewController.removeFromParent()
+    viewController.didMove(toParent: nil)
```

Comment:
Can you refactor your PR so this is the only change? We'd like to keep the project file and `project.yml` as is.

Reply:
✅